Fix autoreconf with >=automake-1.11.2

https://bugs.gentoo.org/397715

Patch written by Kacper Kowalik <xarthisius@gentoo.org>
--- client/Makefile.am
+++ client/Makefile.am
@@ -1,7 +1,7 @@
 INCLUDES = -I$(srcdir)/../services
 
 bin_PROGRAMS = icecc
-pkglib_SCRIPTS = icecc-create-env
+pkgdata_SCRIPTS = icecc-create-env
 icecc_SOURCES = main.cpp arg.cpp cpp.cpp local.cpp remote.cpp util.cpp md5.c safeguard.cpp
 icecc_LDADD = ../services/libicecc.la $(LIBRSYNC)
 noinst_HEADERS = client.h md5.h util.h
